accounts/views.py

```diff
@@ -41,7 +43,13 @@ class SignUpView(generic.CreateView):
 class Home(LoginRequiredMixin, TemplateView):
     template_name = 'accounts/home.html'
 
+    def top_characters(self):
+        return Count.objects.filter(count_type=Count.CHARACTER).all().order_by('-count')[:5] 
+
+    def top_words(self):
+        return Count.objects.filter(count_type=Count.WORD).all().order_by('-count')[:5] 
+
     def dispatch(self, request, *args, **kwargs):
```

Comment:
Heh. I didn't know about that mixin. Must be a newish thing.

Using either the mixin or the decorator, the below code is redundant and should be removed.